nautobot/extras/api/views.py

```diff
@@ -228,6 +229,34 @@ def _get_job_class(self, class_path):
 
         return job_class
 
+    def _create_schedule(self, serializer, data, commit, job, job_class, request):
+        """
+        This is an internal function to create a scheduled job from API data.
+        It has to handle boths once-offs (i.e. of type TYPE_FUTURE) and interval
+        jobs.
+        """
+        job_kwargs = {
+            "data": data,
+            "request": copy_safe_request(request),
+            "user": request.user.pk,
+            "commit": commit,
+            "name": job.class_path,
+        }
+        scheduled_job = ScheduledJob(
+            name=serializer["name"],
+            task="nautobot.extras.jobs.scheduled_job_handler",
+            job_class=job.class_path,
+            start_time=serializer.get("start_time"),
+            description=f"Nautobot job {serializer['name']} scheduled by {request.user} on {serializer.get('start_time')}",
```

Comment:
Can/should we add `description` as an optional field on `NestedJobScheduleSerializer` to allow the user to specify a custom description if desired?

Reply:
Currently we on’t allow this in the UI either. I’d say we should either allow it in both places or neither. What do you think?

Reply:
I'm fine with keeping it as non-user-specifiable for now.